kordac/html-templates/interactive.html

```diff
@@ -0,0 +1,22 @@
+  {% if type == 'in-page' -%}
```

Comment:
Yeah whitespace control, otherwise the newline for the if statements will be present and two empty lines turn into a line break.